src/components/Navbar/NavLanguageSelector.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { CSSProperties } from 'react'
+import { Select } from 'antd'
+import { GlobalOutlined } from '@ant-design/icons'
+
+import { Languages } from 'constants/languages/language-options'
+
+// Language select tool seen in top nav
+export default function LanguageSelector() {
```

Comment:
**praise**: well done on this implementation, and thanks for the detailed comments throughout!

Reply:
Thanks!